libraries/streaming-e2e-tests/tests/streaming-e2e.test.js

```diff
@@ -11,116 +11,144 @@ const userMessage = 'Why hello there';
 const reactAppEndpoint = 'http://localhost:3000';
 
 describe('Chrome', function () {
-  it('should receive an echo after sending a message', async function () {
-    this.timeout(120000);
+    it('should receive an echo after sending a message', async function () {
+        this.timeout(120000);
 
-    const driver = createDriver('chrome');
-    await echoMessageInBrowser(driver);
-    const transcriptMessages = await getTranscriptMessages(driver, 2);
+        const driver = createDriver('chrome');
 
-    await ensureNoBrowserErrors(driver);
-    
-    console.log('Transcript received. Asserting...');
-    assert.strictEqual(transcriptMessages[0], userMessage);
-    assert.strictEqual(transcriptMessages[1], `Streaming Echo: ${userMessage}.`);
-    assert.strictEqual(transcriptMessages.length, 2);
-
-    await driver.quit();
-  });
+        try {
+            await echoMessageInBrowser(driver);
+            await ensureNoBrowserErrors(driver);
+
+            const transcriptMessages = await getTranscriptMessages(driver, 2);
+            await ensureNoBrowserErrors(driver);
+
+            console.log('Transcript received. Asserting...');
+            assert.deepStrictEqual(transcriptMessages, [userMessage, `Streaming Echo: ${userMessage}.`]);
+        } catch (err) {
+            console.log(await driver.takeScreenshot());
```

Comment:
Although it is nice to take a screenshot, this is a Base64 super-long string.

Unless the dev have super high desire and curiosity, probably no one will look at it.

Better save to a file and attached it to the test results (TRX or whatever).

Reply:
Debugging at the moment - will be removing!